wagtail/core/wagtail_hooks.py

```diff
@@ -59,3 +60,19 @@ def register_collection_permissions():
         content_type__app_label='wagtailcore',
         codename__in=['add_collection', 'change_collection', 'delete_collection']
     )
+
+
+@hooks.register('describe_collection_contents')
+def describe_collection_children(collection):
+    descendant_count = collection.get_descendants().count()
+    if descendant_count:
+        url = reverse('wagtailadmin_collections:index')
+        return {
+            'count': descendant_count,
+            'count_text': ngettext(
+                "%(count)s descendant Collection",
+                "%(count)s descendant Collections",
```

Comment:
Better not to capitalise Collection here - this isn't a programmer-facing message, so "2 descendant collections" should be treated as a plain-English concept, like "2 images", rather than as a Python class name.

Reply:
Ah shoot, that's definitely me thinking in programmer-speak. Will fix.